src/components/create-from-builder/create-from-builder-results.html

```diff
@@ -0,0 +1,41 @@
+<div class="center">
+  <div ng-if="!$ctrl.success && !$ctrl.error">
+    <div class="title">
+      Creating Application&hellip;
+    </div>
+  </div>
+  <div ng-if="$ctrl.success">
+    <div class="title">
+      Application Created <span class="fa fa-check success-check"></span>
```

Comment:
> were you planning to update the results screen in a follow-on to be consistent with the updates to bind and order service?

Yeah, and I think order-service still needs updating too. I had planned to make it a follow on.